packaging/macOS/README.md

```diff
@@ -0,0 +1,90 @@
+MacOS installer for Parsec
+==========================
+
+Build steps
+-----------
+
+### 1 - Use a compatible Python version
+
+The app building is done with PyInstaller. For it to work properly, you must use a version of Python anterior to 3.8, otherwise the app won't launch at all. Any 3.6.x or 3.7.x version should work fine.
+
+
+### 2 - Build the application
+
+To make the .app:
+
+```shell
+sh macos_pyinstaller.sh
+```
+
+If a version is already built, you can use the `-f` or `--force` option to bypass the confirmation dialogs when building the new one.
+You can also use the `-i` or `--install` option to directly install the application in the local `/Applications` folder after building.
+
+
+### 3 - Sign the application
+
+```shell
+codesign --deep -s <identity> --timestamp -v --entitlements entitlements.plist -o runtime dist/parsec.app
+```
+
+`identity` being the signing certificate, to be downloaded with proper access here : https://developer.apple.com/account/resources/certificates/list
+
+You can check the codesign using:
+```shell
+codesign -vvv --deep --strict dist/parsec.app
+```
+The next step can only be successful if this check is.
+
+
+### 4 - Zip and Notarize the app
+
+The Apple ID to be used is usually the email address used for the Apple account, and an application-specific password for this script has to be generated here : https://appleid.apple.com/account/manage
```

Comment:
I would be good to explain why this is needed and what is the difference with the signing step.
Typically adding a foreword about Apple step by step release process:
1 - build the app
2 - sign the app
3 - create the notarize .zip file and submit it for natorization
4 - staple the app (I guess it means connecting the notarization result to the signed app... but then what's the point with signing it in the first place ?)

And don't forget to add sarcastic comment about your experience with Apple development :trollface: